src/ClusterBootstrap/ctl.py

```diff
@@ -253,6 +254,67 @@ def render_template_or_dir(config, args):
         utils.render_template(src, dst, config)
 
 
+def maintain_db(config, args):
```

Comment:
Can you put this function into a separate common file so that `deploy.py` can use it? Can you also make it work in `deploy.py`?

Reply:
ok, which file do you prefer? ClusterBootstrap/utils.py?

Reply:
actually I thought about this and would rather keep it in ctl.py, instead of utils.py, as it reads deployment config. if it reads in deployment config, ctl.py should be the file to keep this, and utils.py is mainly for those that is not tangled with configs.
also for deploy.py, since config files are not compatible, it would be faster to convert old config files to fit new pipeline schema